grakn-test/src/main/java/ai/grakn/test/GraknTestEnv.java

```diff
@@ -68,6 +74,19 @@ public static void ensureCassandraRunning() throws Exception {
     }
 
     /**
+     * Create a configuration for use in tests, using random ports.
+     */
+    static GraknEngineConfig createTestConfig() {
+        GraknEngineConfig config = GraknEngineConfig.create();
+
+        Integer serverPort = getEphemeralPort();
```

Comment:
This calls `new ServerSocket(0)` which requests a new "ephemeral" port from the OS, which basically means that it isn't in use.